lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -301,6 +270,7 @@ def replacement_matcher(node)
       #   # good - the above code is rewritten to it by this cop
       #   expect(foo.something?).to be_truthy
       class PredicateMatcher < Cop
+        extend AutoCorrector
```

Comment:
Why is this needed?

Reply:
A cop must `extend AutoCorrector` to be able to autocorrect: https://github.com/rubocop-hq/rubocop/blob/v0.87.1/lib/rubocop/cop/base.rb#L304

Reply:
Oh, I didn't notice this is the actual cop, as it is like 270 lines below the start of the code, though it was a helper class

Reply:
Still somehow confusing, as the offsense/correction is provided by the included modules. Do we need the `autocorrect` method in this class?

Reply:
Shouldn't it be `PredicateMatcher < Base`?

If is it reused, the `module` should probably do it itself:
```
def self.included(base)
  base.extend AutoCorrector
end
```